modules/eco-node-health-check-operator-installation-cli.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/node-health-check-operator-installation.adoc
+
+[id="installing-node-health-check-operator-using-cli_{context}"]
+= Installing the Node Health Check Operator by using the CLI
+You can use the OpenShift CLI (`oc`) to install the Node Health Check Operator.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the Node Health Check Operator:
+.. Define the `Namespace` CR and save the YAML file, for example, `node-health-check-namespace.yaml`:
```

Comment:
I didn't try this method of installation yet so I can't confirm it. @shellymiron may help here